src/ItemsStorageInterface.php

```diff
@@ -7,7 +7,7 @@
 /**
  * A storage for RBAC roles and permissions used in {@see Manager}.
  *
- * @psalm-type ItemsIndexedByName = array<string, Permission|Role>
+ * @psalm-type ItemsIndexedByName = array<string, Item>
```

Comment:
Why? Storage keep permissions and roles, right?

Reply:
`Item` is an abstract class and there are no more items other than permissions and roles. So basically it's equivalent form of writing, but a bit shorter.